dev_guide/compute_resources.adoc

```diff
@@ -302,9 +302,7 @@ limit.
 request.
 |===
 
-A container may have a different quality of service for each compute resource.
-For example, a container can have *Burstable* CPU and *Guaranteed* memory
-qualities of service.
+If a container has requests and limits set that would result in a different quality of service for each compute resource, it will have a *Burstable* QoS Class.
```

Comment:
- QoS Class -> QoS class
- Though, "... it will have a *Burstable* QoS Class" sounds a little funny to me. Maybe instead something like "... it will be classified as *Burstable*." ?

Reply:
Yep, that's a fine suggestion. I agree @bergerhoffer , thanks so much!